src/main.rs

```diff
@@ -34,14 +34,21 @@ fn main_execution() -> Result<bool> {
     let casm = compile(sierra_json)?;
     let casm_json = serde_json::to_value(casm)?;
 
-    let casm_file = OpenOptions::new()
-        .write(true)
-        .create(true)
-        .append(false)
-        .open(args.casm_output_path)
-        .context("Unable to open/create casm json file")?;
+    match args.casm_output_path {
+        Some(output_path) => {
+            let casm_file = OpenOptions::new()
+                .write(true)
+                .create(true)
+                .append(false)
```

Comment:
I think we need `truncate(true)` too
https://stackoverflow.com/questions/65782872/how-to-delete-the-contents-of-a-file-before-writing-to-it